src/controllers/beneficial.owner.individual.controller.ts

```diff
@@ -53,6 +53,7 @@ export const getByID = (req: Request, res: Response, next: NextFunction) => {
 
     return res.render(BENEFICIAL_OWNER_INDIVIDUAL_PAGE, {
       backLinkUrl: BENEFICIAL_OWNER_TYPE_URL,
+      templateName: BENEFICIAL_OWNER_INDIVIDUAL_PAGE,
```

Comment:
need to add the template name into the get function as well for the BO individual controller